ciao-launcher/main.go

```diff
@@ -528,6 +542,14 @@ func startLauncher() int {
 	timeoutCh := make(chan struct{})
 	signal.Notify(signalCh, syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
 
+	configCh := make(chan payloads.Configure)
+	go connectToServer(doneCh, statusCh, configCh)
+	clusterConfig := <-configCh
```

Comment:
@markdryan @sameo, I updated my commit, when you have a chance could you please take a look on it and comment.